projects/uswds-formly/src/lib/types/radio.ts

```diff
@@ -0,0 +1,17 @@
+import { ChangeDetectionStrategy, Component } from '@angular/core';
+
+import { FieldType } from '@ngx-formly/core';
+
+@Component({
+  template: `
+    <usa-radio-group [formControl]="formControl" [name]="to.name ? to.name : 'radio-group'"  [ariaLabelledBy]="to.ariaLabelledBy">
```

Comment:
Also allow `tile` option to be passed in

Reply:
Updated code, will add more demo for it in a separate PR

Reply:
It's OK if there isn't necessarily a demo for every template options available. As long as the user has some way to know what variations exist, that should be OK